packages/trader/src/Modules/Account/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -607,7 +612,7 @@ class PersonalDetailsForm extends React.Component {
                                         has_effect
                                         is_loading={is_btn_loading}
                                         is_submit_success={is_submit_success}
-                                        text={localize('Submit')}
+                                        icon={<Icon icon='IconCheck' />}
```

Comment:
i don't think you need to pass in this icon here, it's already added in the `<Button />` component itself

Reply:
@kelly-binary this is icon managing part in `<Button />` component in deriv-components and `icon` is one of its props.
```jsx
// ...
{icon &&
        <div className='btn__icon'>
            {icon}
        </div>
}
// ...
```
I passed the check-mark icon as a prop to fill this section. I didn't understand what you mean by "it is already added". Can you explain more?

Reply:
@mahdiaryayi-fs yes, `icon` is one of the props. We pass in custom icon to the button through that prop. But, it also has a `is_submit_success` prop. We just need to set it to `true` for it to show the `<IconCheck />` , you can refer to line 614 above 😄

Reply:
you are right :smile: 
I didn't know that. this is my first task on deriv-app! thank you so much for review